mat64/vector.go

```diff
@@ -29,8 +29,9 @@ type Vector struct {
 }
 
 // NewVector creates a new Vector of length n. If len(data) == n, data is used
-// as the backing data slice. If data == nil, a new slice is allocated. If
-// neither of these is true, NewVector will panic.
+// as the underlying slice (changes to the vector will affect the input slice).
```

Comment:
"data is used as the backing slice. ... Changes to elements of the returned Vector will be reflected in data."